examples/ttn-otaa-feather-us915-dht22/ttn-otaa-feather-us915-dht22.ino

```diff
@@ -0,0 +1,248 @@
+/*******************************************************************************
+ * The Things Network - Sensor Data Example
+ * 
+ * Example of sending a valid LoRaWAN packet with DHT22 temperature and
+ * humidity data to The Things Networ using a Feather M0 LoRa.
+ * 
+ * Learn Guide: https://learn.adafruit.com/the-things-network-for-feather
+ * 
+ * Copyright (c) 2015 Thomas Telkamp and Matthijs Kooijman
+ * Copyright (c) 2018 Terry Moore, MCCI
+ * Copyright (c) 2018 Brent Rubell, Adafruit Industries
+ * 
+ * Permission is hereby granted, free of charge, to anyone
+ * obtaining a copy of this document and accompanying files,
+ * to do whatever they want with them without any restriction,
+ * including, but not limited to, copying, modification and redistribution.
+ * NO WARRANTY OF ANY KIND IS PROVIDED.
+ *******************************************************************************/
+#include <lmic.h>
+#include <hal/hal.h>
+#include <SPI.h>
+
+// include the DHT22 Sensor Library
+#include "DHT.h"
```

Comment:
I'm going to want to add this sketch to the CI scripts. That means I'll need to grab the DHT library. Can you give me the URL that matches what you use?